data/beaconrestapi/src/integration-test/resources/tech/pegasys/teku/beaconrestapi/beacon/schema/GetPoolAttesterSlashingsV2Response.json

```diff
@@ -0,0 +1,29 @@
+{
+  "title" : "GetPoolAttesterSlashingsV2Response",
+  "type" : "object",
+  "required" : [ "version", "data" ],
+  "properties" : {
+    "version": {
+      "type": "string",
+      "enum": [
+        "phase0",
+        "altair",
+        "bellatrix",
+        "capella",
+        "deneb",
+        "electra"
+      ]
+    },
+    "data": {
+      "type": "object",
+      "oneOf": [
+        {
+          "type": "array",
+          "items": {
+            "$ref": "#/components/schemas/AttesterSlashing"
+          }
```

Comment:
see if we can get attester slashing for electra to work here too...

also no mention of the `Eth-Consensus-Version` header in the response...

Reply:
Eth-Consensus-Version will be added once the implementation is working. It's prob going to come down to pulling the milestone from one of the AttesterSlashing data.

Reply:
its visible in metadata, we should just add it to the definition so it comes up...

Reply:
actually is it automatic... i cant find it in any gets... hrm...